utils/fix_cmocka_xml.py

```diff
@@ -27,30 +27,39 @@
 # Two pass update on the files
 # Remove all testsuites tag out of the xml file.
 for file in files:
-	if (file != ""):
-		print(file)
-		file_handle =  open('{0}'.format(file), "r+")
-		lines = file_handle.readlines()
-		lines = [tmp.strip(' ') for tmp in lines]
-		while parent_footer in lines : lines.remove(parent_footer)
-		while parent_header in lines : lines.remove(parent_header)
-		file_handle.truncate(0)
-		file_handle.seek(0)
-		file_handle.writelines(lines)
-		file_handle.close()
+    if os.path.isdir(file):
+        print("Skipping %s, it's a directory" % file)
+        continue
+    if file != "":
+        print(file)
+        file_handle =  open('{0}'.format(file), "r+")
+        lines = file_handle.readlines()
+        lines = [tmp.strip(' ') for tmp in lines]
+        while parent_footer in lines:
+            lines.remove(parent_footer)
+        while parent_header in lines:
+            lines.remove(parent_header)
+        file_handle.truncate(0)
+        file_handle.seek(0)
+        file_handle.writelines(lines)
+        file_handle.close()
 #Reconstruct the header and footer
 for file in files:
-	if (file != ""):
-		file_handle =  open('{0}'.format(file), "r+")
-		lines = file_handle.readlines()
-		print(lines[0])
-		if("xml" in lines[0]):
-			lines[0] = lines[0].replace(lines[0],lines[0]+parent_header)
-		else:
-			lines[0] = lines[0].replace(lines[0],xml_header+parent_header+lines[0])
-		file_handle.truncate(0)
-		file_handle.seek(0)
-		file_handle.writelines(lines)
-		file_handle.seek(0,os.SEEK_END)
-		file_handle.writelines(parent_footer)
-		file_handle.close()
+    if os.path.isdir(file):
+        print("Skipping %s, it's a directory" % file)
+        continue
+    if file != "":
```

Comment:
Or this, but it's minor.
```suggestion
    if file == "":
        continue
```